js/crypto.js

```diff
@@ -224,4 +337,14 @@ function loadData() {
     // Hide loading text
     document.getElementById('loading').style.display = 'none';
 
+
+    // events for on change of searchbox input
+
+    document.getElementById('dynamic-coinlist').addEventListener('addItem', function(event) {
+        addCustomCoin(event.detail);
+      }, false);
+    
+    document.getElementById('dynamic-coinlist').addEventListener('removeItem', function(event) {
+        removeCustomCoin(event.detail);
+      }, false);
```

Comment:
Could probably make a var out of `document.getElementById('dynamic-coinlist')`, and `false` needed?

Reply:
_could_ possibly do lots of things, but this seems a sensible suggestion ;)

Reply:
Also,doesn't this work:

document.getElementById('dynamic-coinlist').addEventListener('removeItem', removeCustomCoin);

And then just selecting the `detail` from the `event being passed along to the function